src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -56,7 +59,7 @@ class ReportTypingIndicator extends React.Component {
                         ]}
                         >
                             <Text style={[styles.textStrong]}>{getDisplayName(this.state.usersTyping[0])}</Text>
-                            {' is typing...'}
+                            {this.props.translate('reportTypingIndicator.isTyping')}
```

Comment:
Seems bad to include leading/trailing spaces in the translation. Instead, just translate the phrase, and include leading/trailing spaces in the component:

```suggestion
                            {` ${ this.props.translate('reportTypingIndicator.isTyping')}`}
```